components/https_upgrade_exceptions/browser/BUILD.gn

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+
+static_library("browser") {
```

Comment:
I'm okay with keeping it as `static_library`, because this component is very small and shouldn't grow in future.
relevant discussion: https://bravesoftware.slack.com/archives/C7VLGSR55/p1673461442230819

Reply:
OK, will leave as-is for now. I agree it shouldn't grow.